src/tests/libecalc/input/test_file_io.py

```diff
@@ -158,9 +159,9 @@ def test_valid_characters(self, tmp_path_fixture, csv_line: str, is_valid_charac
             )
 
     def test_missing_headers(self, tmp_path_fixture):
-        with pytest.raises(ValueError) as e:
+        with pytest.raises(InvalidResourceHeaderException) as e:
             file_io.read_facility_resource(create_csv_from_line(tmp_path_fixture, "HEADER1        ,,HEADER3"))
-        assert str(e.value) == "CSV input file must include header"
+        assert str(e.value) == "Missing header(s): One or more headers are missing in time series resource"
```

Comment:
if you had added another test reading from a file or resource, and then calling that function in a test, you would get the filename etc in the error message :)

Reply:
The `create_csv_from_line` actualy gives a filename `csv_file.csv`. It is not captured, as the file name is catched a level above. The only error triggered when using the `file_io.read_facility_resource` isolated like this, is the `InvalidResourceHeaderException` raised in `_validate_headers()`. This does not contain info about file name.